### PR TITLE
fix: make setup easier

### PR DESCRIPTION
**Pull request by lucasmarshall**
Also fixes a few things that were incorrect about how we handle the api url in the nextjs components and sample-app.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,5 +1,3 @@
-NEXT_PUBLIC_SUPAGLUE_HOST=http://localhost:8080
```
Comment (lucasmarshall) [inline .env.sample:1]: This wasn't read by anything.

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ We are currently in Public Alpha. Watch "releases" of this repo to be notified o
 ```shell
 git clone git@github.com:supaglue-labs/supaglue.git
 cd supaglue
-cp .env.sample .env
```
Comment (lucasmarshall) [inline README.md:51]: We have defaults in the docker compose that makes this not nessessary

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -33,12 +33,8 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
-          onlyIncludeVersions: ['0.1.0'],
```
Comment (lucasmarshall) [inline docs/docusaurus.config.js:36]: Show "Next" version of docs.

File: packages/nextjs/src/providers/supaglueProvider.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ const SupaglueContext = createContext({
 
 type SupaglueProviderProps = {
   children: ReactNode;
-  apiUrl: string;
```
Comment (lucasmarshall) [inline packages/nextjs/src/providers/supaglueProvider.tsx:10]: Should be optional since we can fallback to the env var.

**Review by khennes — DISMISSED** [review]

**Pull request merged**